### PR TITLE
Nightly changes

### PR DESCRIPTION
**Pull request by Mav7**

### CODE REVIEW

File: lib/comics_db.rb [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
     asset_names = []
 
-    Dir.entries(assets).each {|a|
+    Dir.entries(assets).each do |a|
       asset_names.push(a) if a == db_table
-    }
+    end
 
     if asset_names.empty?
       false
```
Comment (skippyPeanutButter) [inline lib/comics_db.rb]: It seems a little redundant to return a boolean when the .empty? method itself returns a boolean. In cases like this I prefer to just have the method call as the last line.

So in this case you could say

```
Dir.entries(assets).each do |a|
       asset_names.push(a) if a == db_table
end
 
!asset_names.empty?
```

Comment (Mav7) [inline lib/comics_db.rb]: Ok I got it I'll make the change.

File: lib/comics_dir.rb [hunk 2]
```diff
@@ -67,19 +69,12 @@ def filter_alpha_dirs(dir)
...
 
-  def timestamp(filenames)
-    timestamps = []
-    filenames.each do |e|
-      timestamps.push(File.new(e).ctime)
-    end
-
-    timestamps
-  end
+
```
Comment (skippyPeanutButter) [inline lib/comics_dir.rb]: You can remove this space. Most ruby files tend to have some line of code start right after the class/module declaration and usually the end of a method block before the end of the class/module.

Example: https://github.com/rails/actioncable/blob/master/lib/action_cable.rb

File: lib/comics_db.rb [hunk 3]
```diff
@@ -15,4 +15,5 @@ def self.exists?(db_table, assets)
     end
 
   end
+
```
Comment (skippyPeanutButter) [inline lib/comics_db.rb]: See comment below about spacing

File: spec/comics_dir_spec.rb [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    @comics_dir_aot = ComicsDir.new("Attack on Titan")
+    @comics_dir_mac = ComicsDir.new("mac")
+  end
+
+  describe "#comics_entries" do
+    it ("Returns an array of entries from a directory without previous and current directories.") do
+
+      dir = "./spec/comic_dir/mac"
+      expect(@comics_dir_mac.comics_entries(dir)).not_to(include("..","."))
+
```
Comment (skippyPeanutButter) [inline spec/comics_dir_spec.rb]: Be consistent with your spacing between blocks.

**Pull request closed without merging**